tests/Feature/MorphSnowflakeTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+use Workbench\App\Models\Post;
+use Workbench\App\Models\Tag;
+use Workbench\App\Models\User;
+
+beforeEach(function () {
+    test()->user = User::create(['name' => 'John Doe']);
+    test()->post = Post::create([
+        'title'   => 'My First Post',
+        'user_id' => test()->user->getKey(),
+    ]);
+});
+
+it('creates tags table with morphsSnowflake columns', function () {
+    expect(Schema::hasColumn('tags', 'taggable_id'))->toBeTrue();
+    expect(Schema::hasColumn('tags', 'taggable_type'))->toBeTrue();
+
+    $columns = DB::getSchemaBuilder()->getColumnListing('tags');
+
+    $userTag = new Tag(['name' => 'User Tag']);
+    test()->user->tags()->save($userTag);
+
+    $postTag = new Tag(['name' => 'Post Tag']);
+    test()->post->tags()->save($postTag);
+
+    expect($userTag->refresh()->taggable)->toBeInstanceOf(User::class);
+    expect($postTag->refresh()->taggable)->toBeInstanceOf(Post::class);
+
+    $tags = test()->user->tags;
+    expect($tags)->toHaveCount(1);
+    expect($tags->first()->name)->toBe('User Tag');
```

Comment:
Just a nit, but I generally prefer Pest's higher order expectations:

```suggestion
    expect(test()->user->tags)
        ->toHaveCount(1)
        ->first()->name->toBe('User Tag');
```